### PR TITLE
Create forms

### PR DESCRIPTION
**Pull request by kene-creator**
## Create forms

**Summary**
- Created a method called current_user in your ApplicationController to make current user data available to all controllers.
  - It will return the first user from the database.
- Created forms to perform the following functions:
  - Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
  - Created a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,7 +8,30 @@ def index
   def show
     author_id = posted_by
     @user = User.find(author_id)
-    @post = Post.where(id: params[:id], author_id:).order(id: :DESC).first
+    @post = Post.where(author_id:).first
+    # @post = Post.includes(:comments).where(id: params[:id], author_id:).order('comments.id DESC').first
```
Comment (BertrandConxy) [inline app/controllers/posts_controller.rb]: - [x] Please remove the following commented codes in order to make your codebase clean. This is among the best coding practices that help to code efficiently 😃 👍

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @kene-creator, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (kene-creator) [reply app/controllers/posts_controller.rb]: Thank you

**Review by BertrandConxy — APPROVED** [review]
Hi @kene-creator, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**